rebasehelper/logger.py

```diff
@@ -178,19 +112,90 @@ def emit(self, record):
             self.handleError(record)
 
 
-logging.setLoggerClass(CustomLogger)
-#  the main rebase-helper logger
-logger: CustomLogger = LoggerHelper.get_basic_logger('rebase-helper')
-#  logger for output tool
-logger_output: CustomLogger = LoggerHelper.get_basic_logger('output-tool', logging.INFO)
-logger_report: CustomLogger = LoggerHelper.get_basic_logger('rebase-helper-report', logging.INFO)
-logger_traceback: CustomLogger = LoggerHelper.get_basic_logger('traceback', CustomLogger.TRACE)
-logger_upstream: CustomLogger = LoggerHelper.get_basic_logger('rebase-helper-upstream')
+class LoggerHelper:
+    """Helper class for setting up a logger."""
+
+    @staticmethod
+    def add_stream_handler(logger_object: logging.Logger, level: int = None,
+                           formatter_object: logging.Formatter = None) -> ColorizingStreamHandler:
```

Comment:
I think these `_object` suffixes are unnecessary.

Shouldn't the type here be `Optional[logging.Formatter]`?